07_cmd_lab/cmdlab.txt

```diff
@@ -2,14 +2,26 @@ Intro To Command Line Lab.
 Questions online: https://www.tocode.co.il/bundles/unix1/lessons/cmdlab
 
 1. Login to your UNIX account. How do you find:
-  - your current shell?
+  - your current shell? 
   - your default shell?
   - a list of all shells currently installed on the machine?
 
+
+-echo $0
+-echo $SHELL
+-cat /etc/shells
+
 עברית שפה קשה. בעקרון המעטפת הנוכחית שלי היא bash
 
 2. View the file /etc/passwd on your machine with the command "cat /etc/passwd". Then read about the file in "man 5 passwd". What information is held in the file? What are the fields in each line?
 
+-Contains all users passwords in the following format:
+login name:optional encrypted password:numerical user ID:numerical group ID:user name or comment field:user home directory:optional user command interpreter
+
 3. In Unix, the /etc folder holds configuration files. Display all files in "/etc" folder ordered by file size (Hint: search in "man ls")
 
+ls -s /etc
+
 4. Print current date/time formatted as: "Sun 21/03/2010, 14:30". Hint: read "man date" and search for the correct format string.
+
+date -R
```

Comment:
Didn't work on my machine. Can you paste the output `date -R` printed? It should be formatted as `Sun 23/03/2010 14:30` (i.e. first a day, then the date dd/mm/yyyy and then the time HH:MM)